package.json

```diff
@@ -18,48 +18,48 @@
     "test:build:dist": "jest tasks/gulp/__tests__/after-build-dist.test.js"
   },
   "devDependencies": {
-    "autoprefixer": "^7.1.1",
+    "autoprefixer": "^8.6.4",
```

Comment:
I don't expect anything weird, but with a major bump to autoprefixer should we check it's still outputting what we expect?

Reply:
Good call – checking.

Reply:
there are not output changes in package/ files :) 
from warning text: user-select:none addition